packages/metal-dom/src/keyboardEvents.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+import { object } from 'metal';
+
+export const KEYMAP = {
+  'BACKSPACE': 8,
```

Comment:
Yes, I think Rollup won't help since we use the map to check if the key is valid in the code anyway, so we'll always import the whole object. But, like you said, exporting them separately is still a good option, we can use this kind of import statement when we need an object with all of them.